external/bluetooth/adaptation/include/bt-adaptation-common.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2011 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
Please make 2011 to 2019.

Reply:
These files are external module. and, I didn't change the code.
So, it is better to do not change the boilerplate.